Libraries/Animated/src/NativeAnimatedHelper.js

```diff
@@ -312,10 +316,18 @@ function transformDataType(value: number | string): number | string {
     const radians = (degrees * Math.PI) / 180.0;
     return radians;
   } else {
-    return value;
+    return colorToRgba(value);
   }
 }
 
+function configureProps() {
```

Comment:
Is there a reason for configuring these from JS instead of just having the list live in native implementation of each platform?

Reply:
See prev. comment